tgui-next/packages/tgui/interfaces/MedicalKiosk.js

```diff
@@ -0,0 +1,151 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { AnimatedNumber, Box, Button, LabeledList, ProgressBar, Section} from '../components';
```

Comment:
Missing space after Section and before `}`. Linter can't detect and auto-fix this sadly.